File: ios/Gogai/Utilities/DefaultsKeys.swift
import Foundation

/// UserDefaults のキーを一元管理する（散在ハードコードと衝突を防ぐ）
enum DefaultsKeys {
    /// 旧・未読のみフィルター（Bool）。articleFilter への移行元としてのみ読む
    static let unreadOnly = "unreadOnly"
    static let articleFilter = "articleFilter"
    static let sortOrder = "sortOrder"
    static let serverURL = "serverURL"
    static let resolvedServerURL = "resolvedServerURL"
    static let translationEngine = "translationEngine"
    static let aiProvider = "aiProvider"
    static let stockSortAscending = "stockSortAscending"
    static let stockSummaryQueue = "stockSummaryQueue"
    static let stockSummaryErrors = "stockSummaryErrors"
}

File: android/app/src/main/kotlin/com/mtkg/gogai/cache/DefaultsKeys.kt
package com.mtkg.gogai.cache

/// SharedPreferences のキーを一元管理する（散在ハードコードと衝突を防ぐ、iOS DefaultsKeys の移植）
object DefaultsKeys {
    const val UNREAD_ONLY = "unreadOnly"
    const val SORT_ORDER = "sortOrder"
    const val SERVER_URL = "serverURL"
    const val RESOLVED_SERVER_URL = "resolvedServerURL"
    const val AI_PROVIDER = "aiProvider"
    const val STOCK_SORT_ASCENDING = "stockSortAscending"
    const val STOCK_SUMMARY_QUEUE = "stockSummaryQueue"
    const val STOCK_SUMMARY_ERRORS = "stockSummaryErrors"
}
